src/main/java/useCases/CreateUserAndFridge/CreateUserandFridgeInteractor.java

```diff
@@ -0,0 +1,61 @@
+package useCases.CreateUserAndFridge;
+
+import entities.Fridge.CommonFridge;
+import entities.Fridge.FridgeFactory;
+import entities.Ingrediant.CommonIngredient;
+import entities.user.CommonUser;
+import entities.user.UserFactory;
+import presenters.CreateUserandFridge.CreateUserAndFridgePresenter;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Objects;
+
+public class CreateUserandFridgeInteractor implements CreateUserAndFridgeInputBoundry {
+    UserFactory userFactory;
+    FridgeFactory fridgeFactory;
+
+    CreateUserAndFridgePresenter createUserAndFridgePresenter;
+
+    public CreateUserandFridgeInteractor(UserFactory userFactory, FridgeFactory fridgeFactory, CreateUserAndFridgePresenter
+                                         createUserAndFridgePresenter) {
+        this.userFactory = userFactory;
+        this.fridgeFactory = fridgeFactory;
+        this.createUserAndFridgePresenter = createUserAndFridgePresenter;
+
+    }
+
+    @Override
+    public CreateUserandFridgeResponseModel create(CreateUserandFridgeRequestModel requestModel) {
+        if (Objects.equals(requestModel.getUserName(), "")) {
```

Comment:
Potentially .isEmpty() method might make the code more cleaner - not sure.

Reply:
Changed it